yellowbrick/features/pca.py

```diff
@@ -0,0 +1,103 @@
+##########################################################################
+## Imports
+##########################################################################
+import matplotlib.pyplot as plt
+
+from yellowbrick.features.base import DataVisualizer
+
+from sklearn.pipeline import Pipeline
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+##########################################################################
+##2D PCA Visualizer
+##########################################################################
+class PCA2D(DataVisualizer):
+    """
+    Two dimensional principal component (PC) plot of data projected onto the first and
+    second principal components. It is best practices to center and scale the inputted
+    data set before applying a PC decomposition. There are scale and center arguments
+    that can be used to control centering anc scaling of an inputted data set. Therefore 
+    this class is a one stop shop for easily getting a 2 dimensional PC plot.
+
+    Parameters
+    ----------
+
+    ax : matplotlib Axes, default: None
+        The axes to plot the figure on. If None is passed in the current axes
+        will be used (or generated if required).
+    
+    scale : bool, default: True
+    
+    center : bool, default: True
+    
+    col : list or tuple, default: None
```

Comment:
In other visualizers we call this argument "color" -- e.g. in [ParallelCoordinates](https://github.com/cjmorale/yellowbrick/blob/feature/pca_feature_visualizer/yellowbrick/features/pcoords.py#L59); any reason not to maintain this here?